index.html

```diff
@@ -0,0 +1,276 @@
+                                <img src="./images/github-icon.png" alt="github-icon">
+                            <a/> 
+                        </li>
+                    <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                    </ul>
+                </div>
+                <button class="btn-action" type="button">Get my resume</button> 
+            </div>
+            <div class="skill-set">
+                <div class="tech lang d-flex justify-between align-item-center">
+                    <h4>Languages</h4>
+                    <button type="button">
+                        <i class="fa fa-angle-down downward" aria-hidden="true"></i>
+                    </button>
+                </div>    
+                <ul class="d-flex flex-column no-pad ">
+                    <li class="d-flex align-item-center">
+                        <img src="./images/js-logo.png" alt="js-logo">
+                        <span>javascript</span>
+                    </li>
+                    <li class="d-flex align-item-center">
+                        <img src="./images/html-logo.png" alt="html-logo"> 
+                        <span>HTML</span>
+                    </li>
+                    <li class="d-flex align-item-center">
+                        <img src="./images/css-logo.png" alt="css-logo">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <div>       
+                    <div class="tech d-flex justify-between align-item-center">
+                        <h4>Frameworks</h4>
+                        <button type="button">
+                            <i class="fa fa-angle-right foward" aria-hidden="true"></i>
+                        </button>
+                    </div>
+                
+                
+                    <div class="tech d-flex justify-between  align-item-center">
+                        <h4>Skills</h4>
+                        <button type="button">
+                            <i class="fa fa-angle-right foward" aria-hidden="true"></i>
+                        </button>
+                    </div>
+                </div>   
+            </div> 
+        </section>
+
+        <!-- contact section -->
+        <footer class="contact-me contact-container d-flex flex-column ">
+            <div class="text-center mb-48">
+                <h5 id="contact-me">Contact me</h5>
+                <p id="message">
+                    If you have an application 
+                    you are interested in developing, 
+                    a feature that you need built or 
+                    a project that needs coding. 
+                    I’d love to help with it
+                </p>
+            </div>
+            <form action="https://formspree.io/f/mpzerqgg" method="POST">
+                <div class="d-flex flex-column">
+                    <input type="text" placeholder="Lionel Messi" maxlength="30" required>
```

Comment:
- Kindle add the `name` attribute in your input filed, because the formspree will not submit without the `name` attribute, and its best practice with many usage :+1: 
- [Optional] Great job for using place holder, I think it would be better if you used more miningful discription such as `please enter your name...` :+1: